.github/ISSUE_TEMPLATE/release.md

```diff
@@ -2,17 +2,17 @@
 name: Release Ticket
 about: A ticket tracking each release
 ---
+# Release MM/DD/YYYY
```

Comment:
Despite what the linter says, these have to be second-level since the rendered template appears in a page that already has an H1.
```suggestion
## Release MM/DD/YYYY
```